bin/node/runtime/pangolin/src/pallets/bridge_dispatch.rs

```diff
@@ -0,0 +1,22 @@
+pub use pallet_bridge_dispatch::Instance1 as WithMillauDispatch;
```

Comment:
```suggestion
pub use pallet_bridge_dispatch::Instance1 as WithPangolinDispatchInstance;
```

Reply:
It's right, but can change use same style? `WithMillauDispatchInstance` `WithMillauGrandpaInstance` `WithMillauMessagesInstance`

Reply:
I don't think so.

Checkout these:
https://github.com/darwinia-network/darwinia-common/blob/master/bin/node/runtime/pangolin/src/pallets/collective.rs#L2
https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L526